src/main/java/org/candlepin/pki/impl/JSSProviderLoader.java

```diff
@@ -109,7 +61,7 @@ public static String getJSSVersion() {
      * Initializes the JSS CryptoManager (which in turn initializes the NSS DB), and loads the JSS provider
      * into the JVM. The method should be called during context initialization. Can be called more than once
      * without ill effect (-1 will be returned if the provider is already installed).
```

Comment:
Can we update the last sentence here? This method isn't returning an integer, so it should say something like `the method call will be a no-op` instead if `-1 will be returned`

Reply:
Done.